### DISCUSSION TITLE
Changing the max value after the slider has loaded

### DISCUSSION THREAD

**Discussion by Overbord** [andreruffert/rangeslider.js#365] [Q&A] [answered]
I need to alter the slider dynamically.  I've figured out how to modify valueMapping, but now I need to change the total number of values too.

I've tried just modifying the max value with jQuery, but that didn't work, so I figured something needed to be run after the modification, so I tried running updateOutput with just my new value - updateOutput("11") but that didn't work either.

So I'm pretty stuck now, is it possible?

**Comment by andreruffert** [comment, marked as answer]
👉 https://rangeslider.js.org/#usage

```js
// Update all rangeslider instances for all
// e.g. $('input[type="range"]') elements.
// Usefull if you changed some attributes e.g. `min` or `max` etc.
$('input[type="range"]').rangeslider('update', true);
```

Also this [codepen](https://codepen.io/andreruffert/pen/qdWWZP) for programmatic attribute changes should help.

**Reply by andreruffert** [reply]
@Overbord if this answered your question I'd appreciate if you could mark the discussion as answered. Thanks! ✌️
